### PR TITLE
Created and Updated PRS-IDs

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created and updated a number of PRS-IDs related to BMLacq776. https://github.com/BetaMasaheft/Documentation/issues/2349

### CODE REVIEW

File: PRS3001-4000/PRS3137Cleopatr.xml [hunk 1]
```diff
@@ -35,22 +35,34 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+         <change who="PL" when="2016-03-21"> Created file from google spreadsheet </change>
+         <change who="PL" when="2016-04-28">batch updated according to requirements and issues.</change>
+         <change when="2023-05-03" who="CH">Added names.</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
             <person sex="2">
                <persName>
```
Comment (eu-genia) [inline PRS3001-4000/PRS3137Cleopatr.xml:48]: wd:Q635 and add BM id to wikidata

File: PRS5001-6000/PRS5330Hezekiah.xml [hunk 2]
```diff
@@ -46,8 +47,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <listPerson>
             <person sex="1">
                <persName>
```
Comment (eu-genia) [inline PRS5001-6000/PRS5330Hezekiah.xml:49]: wd:Q244912

and add BM id to wikidata

File: PRS9001-10000/PRS9479Theodosi.xml [hunk 3]
```diff
@@ -44,11 +44,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
-            <person sex="1">
+            <person sex="1" sameAs="wd:Q46696">
```
Comment (eu-genia) [inline PRS9001-10000/PRS9479Theodosi.xml:47]: can you add bm ID to wikidata?

File: new/PRS14018Astaraniqos.xml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="en" xml:id="n1">Aristarchus</persName>
+                    <persName xml:lang="gez" xml:id="n2">ʾAstarāniqos</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n2">አስተራኒቆስ፡</persName>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS14018Astaraniqos" passive="LIT1295Dersan#HomilySane"/>
+                    <relation name="betmas:husbandOf" active="PRS14018Astaraniqos" passive="PRS00000Euphemia"/>
+                    <relation name="betmas:locusTarget" active="#PRS14018Astaraniqos" passive="LOC2177Cilici"/>
```
Comment (eu-genia) [inline new/PRS14018Astaraniqos.xml:55]: remove `#` (actually activity in Cilicia can be put to `floruit`)

File: new/PRS14019Euphemia.xml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+                    <persName xml:lang="gez" xml:id="n3">ʾAfamyā</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n3">አፈምያ፡</persName>
+                    <birth/>
+                    <death/>
+                    <floruit/>
+                </person>
+                <listRelation>
+                    <relation name="betmas:wifeOf" active="PRS14019Euphemia" passive="PRS14018Astaraniqos"/>
+                    <relation name="lawd:hasAttestation" active="PRS14019Euphemia" passive="LIT1295Dersan#HomilySane"/>
+                    <relation name="betmas:locusTarget" active="#PRS14019Euphemia" passive="LOC2177Cilici"/>
```
Comment (eu-genia) [inline new/PRS14019Euphemia.xml:60]: remove `#`

Comment (eu-genia) [reply new/PRS14019Euphemia.xml:60]: actually activity in Cilicia can be put to `floruit`

File: new/PRS14020Ptolemy.xml [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+            </t:langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2023-05-10">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
```
Comment (eu-genia) [inline new/PRS14020Ptolemy.xml:48]: wd:Q39991

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: PRS5001-6000/PRS5330Hezekiah.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply PRS5001-6000/PRS5330Hezekiah.xml:49]: Done.

File: new/PRS14018Astaraniqos.xml (hunk 4 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS14018Astaraniqos.xml:55]: I removed the hash-sign. How can I introduce a LOC-ID to the floruit encounter?
I did not find the word for Cilicia (Qəlqəyā) in the text, but it was mentioned by Marrassini. I don't know for sure, whether Astaraniqos and Afomyo worked or originated in Cilicia - if this makes a difference.

File: PRS3001-4000/PRS3137Cleopatr.xml (hunk 1 above)
Comment (eu-genia) [reply PRS3001-4000/PRS3137Cleopatr.xml:48]: yes, all wikipedia entries - and many more - have wikidata IDs. just search wikidata. and add BM IDs.
https://betamasaheft.eu/Guidelines/?id=wikidata
(e.g. here example for Lebna Dengel 
image
)
Ideally, all named entities that we have should also be on wikidata and linked in both directions.

File: new/PRS14018Astaraniqos.xml (hunk 4 above)
Comment (eu-genia) [reply new/PRS14018Astaraniqos.xml:55]: `<floruit>The hagiographic account places him in <placeName ref="LOC2177Cilici"/></floruit>`

File: PRS3001-4000/PRS3137Cleopatr.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply PRS3001-4000/PRS3137Cleopatr.xml:48]: I think, I need to create an account and log-in. Confusingly, I have the opportunity to edit the page of Lebna Dengel on Wikidata.org, but I cannot change the page of Cleopatra before logging in.

Comment (CarstenHoffmannMarburg) [reply PRS3001-4000/PRS3137Cleopatr.xml:48]: Unless I created an account in wikidata, I cannot edit the page of Cleopatra. For others, as for instance for Ptolemy Auletes, it works.

Comment (CarstenHoffmannMarburg) [reply PRS3001-4000/PRS3137Cleopatr.xml:48]: I added the BM ID to Ptolemy Auletes (wd:Q39991=PRS14020Ptolemy) and to Hezekiah (wd:Q244912=PRS5330Hezekiah). But I was not able to edit the wikidata record for Cleopatra, that is not editable to me.

File: new/PRS14018Astaraniqos.xml (hunk 4 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS14018Astaraniqos.xml:55]: Done for Astaraniqos and for Afomya.

File: PRS3001-4000/PRS3137Cleopatr.xml (hunk 1 above)
Comment (eu-genia) [reply PRS3001-4000/PRS3137Cleopatr.xml:48]: Yes it is protected https://www.wikidata.org/wiki/Talk:Q635 only users with more than 50 edits may edit the page, I am not there either

Comment (CarstenHoffmannMarburg) [reply PRS3001-4000/PRS3137Cleopatr.xml:48]: Do we know somebody with this requirement? Otherwise, we must skip it, don't we?

Comment (eu-genia) [reply PRS3001-4000/PRS3137Cleopatr.xml:48]: I have left a request

File: new/PRS14018Astaraniqos.xml [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+            <change who="CH" when="2023-05-10">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="en" xml:id="n1">Aristarchus</persName>
+                    <persName xml:lang="gez" xml:id="n2">ʾAstarāniqos</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n2">አስተራኒቆስ፡</persName>
```
Comment (thea-m) [inline new/PRS14018Astaraniqos.xml:50]: normalized and fidal swapped

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you! I added the Cleopatra ID to wikidata.

File: new/PRS14018Astaraniqos.xml (hunk 7 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS14018Astaraniqos.xml:50]: Done.

Comment (thea-m) [reply new/PRS14018Astaraniqos.xml:50]: I cannot see the update, is it possible that you corrected this, but did not push the commit? And now I see that the same issue is present in some of the other person records as well. Could you update them as well?

File: new/PRS14018Astaraniqos.xml [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <t:langUsage xmlns:t="http://www.tei-c.org/ns/1.0">
```
Comment (eu-genia) [inline new/PRS14018Astaraniqos.xml:35]: please delete all `t: `and `/t:` and `xmlns:t="http://www.tei-c.org/ns/1.0",` sometimes the app adds the namespace but it should not be there

File: new/PRS14019Euphemia.xml [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <t:langUsage xmlns:t="http://www.tei-c.org/ns/1.0">
```
Comment (eu-genia) [inline new/PRS14019Euphemia.xml:35]: as above

File: new/PRS14020Ptolemy.xml [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <t:langUsage xmlns:t="http://www.tei-c.org/ns/1.0">
```
Comment (eu-genia) [inline new/PRS14020Ptolemy.xml:35]: as above

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS14018Astaraniqos.xml (hunk 7 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS14018Astaraniqos.xml:50]: I did not push the commit, yet. It was a perpetual error. I corrected it for PRS14018Astaraniqos, PRS14019Euphemia, PRS14020Ptolemy, PRS14011GabraMikael and PRS14010Ewostatewos and I will commit in a minute.

File: new/PRS14018Astaraniqos.xml (hunk 8 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS14018Astaraniqos.xml:35]: Done.

File: new/PRS14020Ptolemy.xml (hunk 10 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS14020Ptolemy.xml:35]: Done.

File: new/PRS14020Ptolemy.xml [hunk 11]
```diff
@@ -47,8 +47,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <listPerson>
                 <person sex="1" sameAs="wd:Q39991">
                     <persName xml:lang="en" xml:id="n1">Ptolemy XII Auletes</persName>
-                    <persName xml:lang="gez" xml:id="n2">Baṭlimos</persName>
-                    <persName xml:lang="gez" type="normalized" corresp="#n2">በጥሊሞስ፡ </persName>
+                    <persName xml:lang="gez" xml:id="n2">በጥሊሞስ፡ </persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n2">Baṭlimos</persName>
                     <persName xml:lang="grc" xml:id="n3">Ptolemaios Neos Dionysos Philopatōr Philadelphos</persName>
```
Comment (eu-genia) [inline new/PRS14020Ptolemy.xml:52]: also for Greek you need to switch

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS14020Ptolemy.xml (hunk 11 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS14020Ptolemy.xml]: Done.

**Review by thea-m — APPROVED** [review]
Wonderful, thank you!

File: new/PRS14020Ptolemy.xml [hunk 12]
```diff
@@ -49,10 +49,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
                     <persName xml:lang="gez" xml:id="n2">በጥሊሞስ፡ </persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n2">Baṭlimos</persName>
-                    <persName xml:lang="grc" xml:id="n3">Ptolemaios Neos Dionysos Philopatōr Philadelphos</persName>
-                    <persName xml:lang="grc" type="normalized" corresp="#n2">Πτολεμαῖος Νέος Διόνυσος Φιλοπάτωρ Φιλάδελφος</persName>
-                    <persName xml:lang="grc" xml:id="n4">Ptolemaios Auletes</persName>
-                    <persName xml:lang="grc" type="normalized" corresp="#n2">Πτολεμαῖος Αὐλητής</persName>
+                    <persName xml:lang="grc" xml:id="n3">Πτολεμαῖος Νέος Διόνυσος Φιλοπάτωρ Φιλάδελφος</persName>
+                    <persName xml:lang="grc" type="normalized" corresp="#n2">Ptolemaios Neos Dionysos Philopatōr Philadelphos</persName>
+                    <persName xml:lang="grc" xml:id="n4">Πτολεμαῖος Αὐλητής</persName>
+                    <persName xml:lang="grc" type="normalized" corresp="#n2">Ptolemaios Auletes</persName>
```
Comment (eu-genia) [inline new/PRS14020Ptolemy.xml:55]: it should be `corresp="#n4"`, new PR is needed

Comment (CarstenHoffmannMarburg) [reply new/PRS14020Ptolemy.xml:55]: Done.

**Pull request merged**